modules/apps/transfer/types/packet.go

```diff
@@ -26,6 +27,7 @@ var (
 const (
 	EncodingJSON     = "application/json"
 	EncodingProtobuf = "application/x-protobuf"
+	EncodingABI      = "application/x-solidity-abi"
```

Comment:
I was not able to find an actual "standard" or widely accepted MIME type for solidity ABI encoded data. So I made this one... Let me know if you have a better suggestion.

Reply:
Yeah, I'm not sure what the expectation is here for mime types that don't really exist. 
This would be my hesitation for using mime format rather than just defining our own enum, although, much of a muchness really.. only negative is if at some point a standard mime type is added for abi solidity

Reply:
Yeah, that is true :/ In that case we might just have to support both for a while. Well, this is feedback that @AdityaSripal can think about :)